module/entities/helper.js

```diff
@@ -595,4 +624,39 @@ module.exports = class EntitiesHelper {
 
 }
 
+     /** 
+   * Entities child hierarchy path.
+   * @method
+   * @name childHierarchyPath
+   * @param entityId - entity id.
+   * @returns {Array} Child hierarchy path data.
+  */
+
+   static childHierarchyPath( entityId ) {
```

Comment:
@aman-tunerlabs - Function name will change as per comment in controller

Reply:
@aks30 Need to tell frontend if they accept the changes and make accordingly changes